sync/class.jetpack-sync-actions.php

```diff
@@ -293,6 +294,20 @@ static function add_woocommerce_sync_module( $sync_modules ) {
 		return $sync_modules;
 	}
 
+	static function initialize_wp_super_cache() {
+		//WPCACHEHOME looks like a reasonable constant to test for the plugin's presence as per https://github.com/Automattic/wp-super-cache/blob/master/wp-cache.php
+		if ( false === defined( 'WPCACHEHOME' ) ) {
```

Comment:
Hey @donnchawp - Since we need to check in Jetpack if WP Super Cache is active, we need to find an option, constant, etc. to check against. Is there one that you'd recommend to test against?